pkg/storage/stores/shipper/compactor/table.go

```diff
@@ -90,7 +91,7 @@ func (t *table) compact() error {
 
 	errChan := make(chan error)
 	readObjectChan := make(chan string)
-	n := util.Min(len(objects), readDBsParallelism)
+	n := math.Min(len(objects), readDBsParallelism)
```

Comment:
Would you mind making these consistently `util_math`?

Reply:
👍